web-client/src/app/utils/errors/global-error-handler.ts

```diff
@@ -19,6 +19,25 @@ export class GlobalErrorHandler implements ErrorHandler {
       return;
     }
 
+    if (
+      error.message.includes('SessionService.signTransaction: invalid auth')
+    ) {
+      console.warn('GlobalErrorHandler:', error);
+      this.zone.run(
```

Comment:
I was trying to copy how we threw an error notification for "Unexpected Error Occurred".

I think we're using zone so that we can use async functions.

If not, can you recommend how I should refactor this?
It's also difficult to move this notification to `session.service.ts` because then the functions return type expects a promise etc.

Reply:
I don't think ngZone is necessary but it doesn't do harm either.
we can have a quick session after the planning to see what can be done

Reply:
I caught it somewhere else and threw a Local Error and ignored the local error in the global error handler :)

Reply:
@billguo99 can you just add a TODO to revisit and refactor later

Reply:
Done :+1: